test/suites-ostree/release-y-stream-upgrade-and-rollback.robot

```diff
@@ -0,0 +1,39 @@
+*** Settings ***
```

Comment:
Work is currently stuck on failures in the build phase.  This file was copied from another, but hasn't been touched yet.